EXPERIMENTAL.org

```diff
@@ -52,11 +53,28 @@ your OS).
 In the root directory of your freshly cloned Emacs repository and with the
 =pdumper= branch as the current branch:
 
+#+BEGIN_SRC shell
+  ./autogen.sh
+  ./configure --with-dbus --with-gnutls --with-imagemagick --with-rsvg  --with-mailutils --with-xml2 --with-modules --prefix="${HOME}/local"
+#+END_SRC
+
+**** Ubuntu
+In the root directory of your freshly cloned Emacs repository and with the
+=pdumper= branch as the current branch:
+
 #+BEGIN_SRC shell
   ./autogen.sh
   ./configure --with-ns --with-dbus --with-gnutls --with-imagemagick --with-rsvg  --with-mailutils --with-xml2 --with-modules
 #+END_SRC
 
+If you have never compile Emacs from source on your machine then you probably
+need to install the following packages:
+
+#+BEGIN_SRC shell
+   sudo apt-get install build-essential automake texinfo libjpeg-dev libncurses5-dev
+   sudo apt-get install libtiff5-dev libgif-dev libpng-dev libxpm-dev libgtk-3-dev libgnutls28-dev
```

Comment:
Better command to install almost all the necessary libraries:
```
apt-get build-dep emacs
```
I don't know exactly which packages aren't covered by it, though.

Reply:
I don't think that actually works. I assume  the command  will look for dependencies of deb package `emacs` right? That will point to Emacs 25.

Reply:
right, but it still covers most of the dependencies IIRC

Reply:
Last time I checked, the current `emacs` in Ubuntu's repo is Emacs 25. I am not sure building dependencies for Emacs 25 would be very helpful since this is Emacs 27.

Reply:
I don't have an Ubuntu machine myself to check exactly what libraries are installed, but I do expect it to be very helpful for Emacs 27 as well. Anyway, I'm not going to insist on changing the instructions. If you believe current instructions are good enough, lets roll with it.